### ISSUE TITLE
[MAS1.4.3] [Visual Requirements- Add language understanding] Luminosity Ratio is less than 4.5:1  for “add a LUIS app manually “ link present in the LUIS application  dialog

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1798]
**Actual Result:**
Luminosity Ratio is less than 4.5:1  for “add a LUIS app manually “ link present in the LUIS application  dialog

**Expected Result:**
Luminosity Ratio should be greater than or equal to 4.5:1 for “add a LUIS app manually “ link

**Note:** Same issue repro for below path
Bot Explorer->Services->+symbol->Add QnA maker
Please refer the attachment:MAS1 4 3_Qna maker.png
Bot Explorer->Services->+symbol->Add Dispatch
Please refer the attachment:MAS1 4 3_Add a Dispatch.png

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to the bot explorer and select it 
3.Navigate to the services symbol and select it 
4.Navigate to the Add Language understanding(LUIS) and select it 
5.Dialog gets opened 
6.Open 'Colour Contrast Analyser' application.
7.Verify whether Luminosity Ratio is less than 4.5:1 for “add a LUIS app manually “or not 

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of link is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Tool: Colour contrast analyser

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![MAS1 4 3_Add language understanding](https://user-images.githubusercontent.com/53518994/64019954-131bfe00-cb4e-11e9-9122-5f6b2babc251.png)

![MAS1 4 3_Qna maker](https://user-images.githubusercontent.com/53518994/64020016-38a90780-cb4e-11e9-998f-a1330f01de85.png)
![MAS1 4 3_Add a Dispatch](https://user-images.githubusercontent.com/53518994/64020051-478fba00-cb4e-11e9-8bb0-72d177b8797a.png)

**Comment by tonyanziano** [comment]
This was fixed in #1828 

![image](https://user-images.githubusercontent.com/3452012/64388186-2cf88d80-cff3-11e9-99c3-90d6119becce.png)

![image](https://user-images.githubusercontent.com/3452012/64388233-4f8aa680-cff3-11e9-8dc0-f727ea745f4f.png)

![image](https://user-images.githubusercontent.com/3452012/64388277-6cbf7500-cff3-11e9-9d44-4e369514ed39.png)
